examples/embeddable_examples/public/app/presentation_container_example/types.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { TimeRange } from '@kbn/es-query';
+import {
+  CanAddNewPanel,
+  HasSerializedChildState,
+  HasRuntimeChildState,
+  PresentationContainer,
+  SerializedPanelState,
+  HasSaveNotification,
+} from '@kbn/presentation-containers';
+import {
+  PublishesDataLoading,
+  PublishesPanelTitle,
+  PublishesTimeRange,
+  PublishesUnsavedChanges,
+  PublishesViewMode,
+} from '@kbn/presentation-publishing';
+import { PublishesReload } from '@kbn/presentation-publishing/interfaces/fetch/publishes_reload';
+
+export type ParentApi = PresentationContainer &
+  CanAddNewPanel &
+  HasSaveNotification &
+  HasSerializedChildState &
+  HasRuntimeChildState &
+  PublishesDataLoading &
+  PublishesViewMode &
+  Pick<PublishesPanelTitle, 'hidePanelTitle'> &
```

Comment:
That parent API doesn't do anything with the `hidePanelTitle`, does it? Do we want to clean up this API to only include the parts that this example actually uses?

Reply:
I am not sure why this was added to the API. I removed it in e95bf979466223968cdb90a7acf89bf11e124915. The settings UI needs to be updated to not show the "Show titles" toggle but that is outside the scope of this PR